examples/ucp_hello_world.c

```diff
@@ -216,11 +216,11 @@ static void close_ep(ucp_ep_h ep)
 {
     ucp_request_param_t ep_close_param;
 
+    ep_close_param.op_attr_mask = UCP_OP_ATTR_FIELD_FLAGS;
     if (err_handling_opt.ucp_err_mode == UCP_ERR_HANDLING_MODE_PEER) {
-        ep_close_param.op_attr_mask = UCP_OP_ATTR_FIELD_FLAGS;
-        ep_close_param.flags        = UCP_EP_CLOSE_FLAG_FORCE;
+        ep_close_param.flags = UCP_EP_CLOSE_FLAG_FORCE;
     } else {
-        ep_close_param.op_attr_mask = 0;
+        ep_close_param.flags = 0;
```

Comment:
why did it fail? if op_attr_mask is 0 it means flag value should be assumed as 0

Reply:
This changes was irrelevant for the issue.
The next commit solved the issue.